TAGS

```diff
@@ -1,5 +1,5 @@
-8.11:8.11.0:8.11 8 latest
-8.11/slim:8.11.0-slim:8.11-slim 8-slim latest-slim
+8.11:8.11.1:8.11 8 latest
+8.11/slim:8.11.1-slim:8.11-slim 8-slim latest-slim
```

Comment:
Shall we stop refreshing 5.x, 6.x and 7.x already now? Then they will fall off the "supported" list. The tags will still be available, but not promoted.

Reply:
I suggest to have this debate after 8.11.1 is released because open ended questions during a security update just delays action.  Even if it's mitigated, it'd be nice to get the vulnerable Log4J out of a build.  Security people all over the place are watching and blocking things that throw red flags even if they're 'mitigated'.

Reply:
Yes, we can fix #337  separately, but it is unfortunate to pretend that 5 year old versions are "supported"

Reply:
_Right_ now doesn't seem the time to do that.